var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -22,6 +20,13 @@
 # License along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
+#
+# Notes
+# - swr: need better handling of C++14 constructs
+#        may need to export LD_LIBRARY_PATH for dependent packages
```

Comment:
Was RPATH not sufficient in some case for you when a Spack-built package depends on this?

Reply:
I haven't seen any problems related to LD_LIBRARY_PATH in my testing. This comment may come from one of the prior package versions that was merged by @olesenm. I think it can be ignored, and I will remove it.

Reply:
The comment about C++14 is still relevant, because some features of mesa require a compatible compiler. Unfortunately, the mesa configure script sometimes activates features that cannot be built with the compiler being used, which causes the entire build to fail. We handle these cases by checking the gcc version and enabling or disabling the mesa features explicitly. That method works with gcc and known versions of mesa, but it could break in future with different compilers or versions of mesa. The comment provides a hint that could be useful in diagnosing such problems.

Reply:
If I understand correctly you are removing the part about LD_LIBRARY_PATH but keeping the part about C++14? If so IMO the comment you added on line 107 captures the C++14 part.

Reply:
OK, I will remove the redundant C++14 comment too.